pkg/devices/coils.go

```diff
@@ -114,21 +131,24 @@ func bulkReadCoils(managers []*ModbusDeviceManager) ([]*sdk.ReadContext, error)
 					if manager.FailOnError {
 						return nil, err
 					}
-					log.WithError(err).Warning("ignoring error on register unpack (failOnError is false)")
+					log.WithError(err).Warning("[modbus] ignoring error on register unpack (failOnError is false)")
 					continue
 				}
 
 				log.WithFields(log.Fields{
 					"device": device.Device.GetID(),
 					"type":   reading.Type,
 					"value":  reading.Value,
-				}).Debug("created new reading from input register")
+				}).Debug("[modbus] created new reading from input register")
 				readCtx := sdk.NewReadContext(device.Device, []*output.Reading{reading})
 				readings = append(readings, readCtx)
 			}
 		}
 	}
 
+	if managerErrCount == len(managers) {
```

Comment:
Which hosts failed would be useful in the log. (here and when reading registers)

Reply:
Good call. Updated in most recent commit.